packages/kbn-unified-data-table/src/components/data_table.tsx

```diff
@@ -716,6 +722,10 @@ export const UnifiedDataTable = ({
     [defaultColumns, isSortEnabled, additionalControls, showDisplaySelector, showFullScreenButton]
   );
 
+  const gridStyle = useMemo(() => {
+    return gridStyleOverride ?? GRID_STYLE;
+  }, [gridStyleOverride]);
+
```

Comment:
Generally, yes memoization is very important to the grid's usage, we have to omit re-renderings, since they are expensive. However in this case I'm curious, what's the benefit of this memoization? GRID_STYLE is a constant, and gridStyleOverride should also be (unless the consumer uses it like this `gridStyleOverride={{stripes: false}}`, but then useMemo like here wouldn't help)

Reply:
you're right on that, I was considering the last example usage, but in fact, `useMemo` wouldn't give any benefit at all since the checking it does would end up being more expensive than just checking if the variable exists